src/test/java/com/google/javascript/clutz/partial/missing_type.d.ts

```diff
@@ -0,0 +1,10 @@
+declare namespace ಠ_ಠ.clutz.module$exports$missing$type {
+  var x : any ;
```

Comment:
yep, one PR at a time. I am unrolling a bunch of local changes I have internally. That would be the next PR.